views/make/settings.html

```diff
@@ -10,7 +10,8 @@
     </div>
     <div class="icon-container">
         <div style="background: {{app.iconColor}}" class="app-icon">
-          <div class="icon-image" style="background-image:url('{{app.iconImage}}')"></div>
+
+          <div class="icon-image" id="icon-{{$index}}" v-repeat="icon: iconImages" v-show="$index === currentIconIndex" v-transition style="background-image:url('{{icon}}')"></div>
```

Comment:
Maybe give `v-style="background-image: 'url(' + {{icon}} + ')'"` (untested...just FYI :smile_cat:)

For reference: http://vuejs.org/api/directives.html#v-style